components/tabstrip/persist-content.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Persist Content
+page_title: Persist Content
+description: Persist Content of the TabStrip for Blazor.
+slug: tabstrip-persist-content
+tags: telerik,blazor,tab,strip,tabstrip,overview
+published: True
+position: 10
+---
+
+# Persist Content
+
+By default, the content of the TabStrip is rendered in the DOM when the Tab is active. Once it loses focus, its content is disposed and re-initialized again when the user selects the corresponding tab.
```

Comment:
deactivated, loss of focus implies something like blurring an input inside it

Reply:
Updated